internal/agent/collector/vsphere.go

```diff
@@ -315,6 +315,9 @@ func FillInventoryObjectWithMoreData(vms *[]vspheremodel.VM, inv *apiplanner.Inv
 	inv.Vms.DiskCount.Histogram = Histogram(diskCountSet)
 	inv.Vms.DiskGB.Histogram = Histogram(diskGBSet)
 	inv.Vms.NicCount.Histogram = Histogram(nicCountSet)
+
+	// Update disk size tier
+	updateDiskSizeTier(diskGBSet, inv)
```

Comment:
Just a thought (not sure about this)
Maybe better just save the totalCapacity for each VM in the inventory and than do the calculation in the Service or in the UI?
This way we will be able in the future to use the same info for different calculation

Reply:
@machacekondra any thoughts here?

Reply:
We can do it but it means changing some structures here and not related directly to the PR
If you think it's needed, let's open a relevant ticket for that